### PR TITLE
Initializing MongoDB and creating ReviewController which can read and write to the review collection

### PR DESCRIPTION
**Pull request by NickPriv**

### CODE REVIEW

File: src/main/java/com/fryrank/controller/ReviewController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import java.util.List;
+
+@RestController
+public class ReviewController {
+
+    @Autowired
+    private ReviewDAL reviewDAL;
+
+    @GetMapping(value = "/reviews/")
```
Comment (oxyflush) [inline src/main/java/com/fryrank/controller/ReviewController.java:17]: Not sure about standards, but do RESTful URL paths usually end in `/`?

File: pom.xml [hunk 2]
```diff
@@ -20,7 +20,6 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
-			<version>2.1.2.RELEASE</version>
```
Comment (oxyflush) [inline pom.xml:23]: Just for my own clarity, why was 2.1.2 RELEASE not a good version? What's being used now?

**Review by oxyflush — APPROVED** [review]
## :thumbsup: Overall LGTM

File: src/main/java/com/fryrank/controller/ReviewController.java (hunk 1 above)
Comment (NickPriv) [reply src/main/java/com/fryrank/controller/ReviewController.java:17]: Yeah I think you're right. Will fix that

File: pom.xml (hunk 2 above)
Comment (NickPriv) [reply pom.xml:23]: There was a build conflict when booting up the application. Now it is using 3.1.0 (according to IntelliJ dependencies list) which is a newer version.

**Pull request merged**